app/Grouptopics/Repositories/Account/AccountEloquent.php

```diff
@@ -1,19 +1,22 @@
 <?php
-namespace Grouptopics\Libraries;
+namespace Grouptopics\Repositories\Account;
 
-use \Input as Input;
-use \Auth as Auth;
-use \User as User;
-use \Validator as Validator;
-use \Hash as Hash;
-use \Redirect as Redirect;
+use Grouptopics\Repositories\EloquentRepository;
 
-class Account
+
+class AccountEloquent extends EloquentRepository implements AccountInterface
```

Comment:
I use Eloquent as the naming convention so that you know its using the Eloquent Libraries. I like to identify that in the name.